lib/matplotlib/category.py

```diff
@@ -85,7 +85,7 @@ def axisinfo(unit, axis):
         `~matplotlib.units.AxisInfo`
             Information to support default tick labeling
 
-        .. note: axis is not used
+        .. note:: *axis* is not used
```

Comment:
added `*` to match line 42.

Reply:
This should move under the `axis` parameter docs, like on line 42, and not be in the Returns section.

Reply:
Sure. Done.